masterlist.yaml

```diff
@@ -28,10 +28,84 @@ globals:
         text: '[Postagem mais recente do LOOT](%1%).'
     subs: [ 'https://loot.github.io/latest-thread/' ]
 
-groups: []
+groups:
+- name: &dlcGroup DLC
+
+- name: &unofficialPatches Unofficial Patches
+  after:
+  - *dlcGroup
+
+- name: &default Default
+  after: 
+  - *unofficialPatches
+
+- name: &dynamicPatchGroup Dynamic Patches
+  after:
+  - *default
 
 plugins:
 # Official DLC
+  - name: 'Tribunal.esm'
+    group: *dlcGroup
+
   - name: 'Bloodmoon.esm'
+    group: *dlcGroup
     after:
       - 'Tribunal.esm'
+      
+# Unofficial Patches
+
+  - name: 'Patch for Purists - Dialogue Typos.esm'
+    group: *unofficialPatches
+
+  - name: 'Patch for Purists - Base Patch.esm'
+    group: *unofficialPatches
+    after:
+      - name: 'Patch for Purists - Dialogue Typos.esm'
+    msg:
+      - type: error
+        condition: 'active("Morrowind Rebirth [Main].ESP")'
+        content:
+          - lang: en
+            text: 'Morrowind Rebirth is not compatible with Patch of Purists. Instead, "Morrowind Patch Project 1.6.5 [For Rebirth].esm" which comes with it should be used.'
+      - type: error
```

Comment:
Why not use [`inc` metadata](https://loot-api.readthedocs.io/en/latest/metadata/data_structures/plugin.html) for this message and the one after? E.g.

```
- name: 'Patch for Purists - Base Patch.esm'
  inc:
    - gr_ScriptImprovements.esp
    - gr_ScriptImprovements.esm
    - Morrowind Patch v1.2.2.esm
    - Morrowind Patch v1.6.3.esm
    - Morrowind Patch v1.6.4.esm
    # etc...
```

Reply:
I could, but I wanted custom messages for each. That's not possible with inc, right?

Reply:
Yes, that's true.